src/app/app.component.scss

```diff
@@ -0,0 +1,6 @@
+h1 {
```

Comment:
A know that there is only one `h1` element on this site but it's a good practice to style elements by class, not by tag. So maybe class `.header` should stay and it should be used here.

Reply:
Done

Reply:
That's good idea. Done